test/forge/integration/AccrueInterestIntegrationTest.sol

```diff
@@ -90,8 +97,7 @@ contract AccrueInterestIntegrationTest is BaseTest {
         morpho.supplyCollateral(marketParams, 1, address(this), hex"");
         vm.expectEmit(true, true, true, true, address(morpho));
         emit EventsLib.AccrueInterest(id, borrowRate, expectedAccruedInterest, 0);
-        // Accrues interest.
-        morpho.withdrawCollateral(marketParams, 1, address(this), address(this));
+        morpho.accrueInterest(marketParams);
```

Comment:
We can also remove the lines 96 and 97, they are not needed anymore since accruing interest does not require to supplyCollateral before withdrawCollateral